File-Formats/XlsIO/Loading-and-Saving/loading-and-saving-excel-files-in-maui-c-sharp.md

```diff
@@ -59,6 +59,14 @@ workbook.SaveAs(outputStream);
 outputStream.Position = 0;
 
 //Saves the memory stream as a file.
-DependencyService.Get<ISave>().SaveAndView("Output.xlsx", "application/excel", outputStream);
+SaveService saveService = new SaveService();
+saveService.SaveAndView("Output.xlsx", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", ms);
 {% endhighlight %}
 {% endtabs %} 
+
+The code for SaveService helper class is present in below pages.
```

Comment:
can we import the code from getting started instead links similar to Blazor.

Reply:
@Mohan2401 - Made changes as suggested